### PR TITLE
Make the README a bit slimmer

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**
* Add fancy links and a new badge to the top of the README
* Document sending directly to Splunk ingest

### CODE REVIEW

File: docs/middleware-attributes.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Middleware Attributes
```
Comment (mateuszrzeszutek) [inline docs/middleware-attributes.md:1]: Middleware docs will be added in another PR

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Troubleshooting
+
+To turn on the agent's internal debug logging:
+
+`-Dotel.javaagent.debug=true`
+
+> :warning: Debug logging is extremely verbose and resource intensive. Enable
+> debug logging only when needed and disable when done.
```
Comment (mateuszrzeszutek) [inline docs/troubleshooting.md:8]: More troubleshooting docs will be added in another PR

File: docs/advanced-config.md [hunk 3]
```diff
@@ -23,6 +23,15 @@ Below you will find all the configuration options supported by this distribution
 | `otel.traces.exporter`          | `OTEL_TRACES_EXPORTER`            | `jaeger-thrift-splunk`           | Select the span exporter to use.
 | `otel.exporter.jaeger.endpoint` | `OTEL_EXPORTER_JAEGER_ENDPOINT`   | `http://localhost:9080/v1/trace` | The Jaeger endpoint to connect to.
 
+The Jaeger exporter can be configured to export traces directly to Splunk ingest.
+To achieve that, you need to set the `splunk.access.token` configuration property
+and set the `otel.exporter.jaeger.endpoint` to Splunk ingest URL. For example:
+
+```bash
+export SPLUNK_ACCESS_TOKEN=my_splunk_token
+export OTELK_EXPORTER_JAEGER_ENDPOINT=https://ingest.us0.signalfx.com/v2/trace
```
Comment (iNikem) [inline docs/advanced-config.md:32]:
```suggestion
export OTEL_EXPORTER_JAEGER_ENDPOINT=https://ingest.us0.signalfx.com/v2/trace
```

File: docs/correlating-traces-with-logs.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Correlating traces with logs
+
+The Java Agent injects several pieces of metadata into the logger MDC.
+You can use the following information to correlate traces with logs:
+
+- Trace information: `trace_id` and `span_id`;
+- Resource attributes: `service.name` and `deployment.environment`.
+
+Injecting trace context information is described in detail on
+[this page](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md).
```
Comment (iNikem) [inline docs/correlating-traces-with-logs.md:10]:
```suggestion
[this page](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/logger-mdc-instrumentation.md).
```

File: docs/correlating-traces-with-logs.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# Correlating traces with logs
+
+The Java Agent injects several pieces of metadata into the logger MDC.
+You can use the following information to correlate traces with logs:
+
+- Trace information: `trace_id` and `span_id`;
+- Resource attributes: `service.name` and `deployment.environment`.
+
+Injecting trace context information is described in detail on
```
Comment (iNikem) [inline docs/correlating-traces-with-logs.md:9]:
```suggestion
Injecting trace context is described in detail on
```

**Review by iNikem — APPROVED** [review]
